### PR TITLE
Add some love to the installer

### LINKED ISSUE #1717: Installer UI could use some love.
## Type
- Enhancement

### PR DESCRIPTION
**Pull request by Katrienvh**
## Type
- Enhancement

## Resolves the following issues
fixes #1717 

## Pull request description
Give the installer layout some love so it feels/looks more like Fork CMS backend.
Added a progress bar, so you know where in the install process you are.

screen shot 2018-01-25 at 11 38 08

### CODE REVIEW

File: src/ForkCMS/Bundle/InstallerBundle/Resources/views/Installer/base.html.twig [hunk 1]
```diff
@@ -24,20 +24,48 @@
   {% endjavascripts %}
 </head>
 <body id="installer">
-<div id="genericBox" class="container">
-  <div class="row">
-    <div class="col-md-10 col-md-offset-1">
-      <div id="genericBoxTitle" class="page-header text-center">
-        <h2>Install Fork CMS</h2>
+<div id="root">
+  <div id="genericBox">
```
Comment (StijnVrolijk) [inline src/ForkCMS/Bundle/InstallerBundle/Resources/views/Installer/base.html.twig]: I believe the genericBox id's are useless, they're not used in any CSS or JS files

File: src/ForkCMS/Bundle/InstallerBundle/Resources/views/Installer/base.html.twig [hunk 2]
```diff
@@ -24,20 +24,48 @@
...
+    <div class="header">
+      <div class="container">
+        <div class="row">
+          <div class="col-md-10 col-md-offset-1">
+            <div id="genericBoxTitle" class="text-center">
```
Comment (StijnVrolijk) [inline src/ForkCMS/Bundle/InstallerBundle/Resources/views/Installer/base.html.twig]: Same here

File: src/ForkCMS/Bundle/InstallerBundle/Resources/views/Installer/navigation.html.twig [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      <li{% if step == 'step-5' %} class="active"{% endif %}>
+        Step 5
+      </li>
+      <li{% if step == 'step-6' %} class="active"{% endif %}>
+        Step 6
+      </li>
+    </ul>
+    <p class="hidden-md hidden-lg">
+      Step
+      {% if step == 'step-1' %}
```
Comment (StijnVrolijk) [inline src/ForkCMS/Bundle/InstallerBundle/Resources/views/Installer/navigation.html.twig]: You can just use `{{ step|replace({'step-': ''}) }}` to display the step

Comment (carakas) [reply src/ForkCMS/Bundle/InstallerBundle/Resources/views/Installer/navigation.html.twig]: smart idea

**Pull request merged**